src/openrct2-ui/scripting/ScUi.hpp

```diff
@@ -388,6 +406,7 @@ namespace OpenRCT2::Scripting
             dukglue_register_method(ctx, &ScUi::registerMenuItem, "registerMenuItem");
             dukglue_register_method(ctx, &ScUi::registerToolboxMenuItem, "registerToolboxMenuItem");
             dukglue_register_method(ctx, &ScUi::registerShortcut, "registerShortcut");
+            dukglue_register_method(ctx, &ScUi::setDropdownColours, "setDropdownColours");
```

Comment:
This should be added to `openrct2.d.ts`. Is looks like when an empty array is passed, it'll revert to the default colours. Wouldn't a separate function for this behaviour be clearer? What happens if a plug-in passes `null` or `undefined`?